app/src/models/business.js

```diff
@@ -14,17 +14,44 @@ module.exports = (sequelize, DataTypes) => {
       comment: 'The business name',
       type: DataTypes.STRING(255),
       unique: false
+    },
+    addressLine1: {
+      allowNull: false,
+      comment: 'Address line 1',
+      type: DataTypes.STRING(255),
+      unique: false
+    },
+    addressLine2: {
+      allowNull: false,
```

Comment:
AddressLine2 is optional (unless we're just writing empty string if it's not supplied)

Reply:
good catch..

Reply:
fixed

Reply:
fixed